packages/api/src/webhooks/cannon.ts

```diff
@@ -222,6 +239,69 @@ export default class WebhookCannon {
     );
   }
 
+  async notifyDisabledWebhook(
+    trigger: messages.WebhookTrigger,
+    params?: any,
```

Comment:
The type here should be `RequestInit & { timeout?: number }` (like this one https://github.com/livepeer/livepeer-com/blob/a21b96ad54399ce8694e32e013dc2414b6b16bce/packages/api/src/util.ts#L57)

Also in the `retry` function that calls this.